app/app.js

```diff
@@ -41,19 +42,24 @@ angular.module('app', [])
     $expandedComic.css('marginBottom', '');
 
     if (vm.expandedComic === currentComic.id) {
-      vm.expandedComic = undefined;
+      vm.expandedComic      = undefined;
+      vm.expandedCollection = undefined;
```

Comment:
Is it good practice to align the assign = ?

Reply:
It's a personal preference of mine to do it with like variables as a visual indication that they're grouped in some way.